docs/configuration/human-readable-byte.md

```diff
@@ -0,0 +1,88 @@
+---
+id: human-readable-byte
+title: "Human-readable Byte Configuration Reference"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+
+This page documents all configuration properties related to bytes.
+
+These properties can be configured through 2 ways: 
+1. a simple number in byte
+2. a number with a unit suffix
+
+## A number in byte
+
+Given the cache size is 3G, there's a configuration as below
+
+```properties
+# 3G bytes = 3_000_000_000 bytes
+druid.cache.sizeInBytes=3000000000 
+```
+
+
+## A number with a unit suffix
+
+Sometimes value in byte could be very large in Druid, it's counterintuitive to set value of these properties as above.
+Here comes another way, a number with a unit suffix.
```

Comment:
Hmm.. I'm not sure whether or not `counterintuitive` is being used correctly here. AFAIK, it means something which doesn't seems true intuitively but is actually true. Maybe it would be more clear by rephrasing this paragraph to `When you have to put a large number for some configuration, it is easy to make a mistake such as extra or missing 0s. Druid supports a better way, a number with a unit suffix.`